content/topics/ncit/114048-and-114049/_index.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Database Development  (US 114048 & 114049)
+ready: true
+tags: ["ncit", "sql"]
```

Comment:
This would be better I think:

```
---
title: Database Development
ready: true
tags: ["sql"]
nqf: ncit    # implying this is a requirement for ncit
unit_standards: [114048, 114049]
---
```

Reason being: It's better to have things ultra clear and specific and seperate when they mean different things. It's ok to add new kinds of data to the frontmatter.

Reply:
@sheenarbw I made the changes